activiti-cloud-services-runtime-bundle/activiti-cloud-services-connectors/src/main/java/org/activiti/services/connectors/behavior/MQServiceTaskBehavior.java

```diff
@@ -38,45 +46,62 @@
     private final IntegrationContextManager integrationContextManager;
     private final ApplicationEventPublisher eventPublisher;
     private final IntegrationContextBuilder integrationContextBuilder;
+    private final List<ConnectorDefinition> connectorDefinitions;
+    private final ConnectorActionDefinitionFinder connectorActionDefinitionFinder;
+    private final VariablesMatchHelper variablesMatchHelper;
     private final RuntimeBundleInfoAppender runtimeBundleInfoAppender;
 
     public MQServiceTaskBehavior(IntegrationContextManager integrationContextManager,
                                  ApplicationEventPublisher eventPublisher,
                                  ApplicationContext applicationContext,
                                  IntegrationContextBuilder integrationContextBuilder,
+                                 List<ConnectorDefinition> connectorDefinitions,
```

Comment:
If you pass the connector list directly to the finder (https://github.com/Activiti/Activiti/pull/1984#discussion_r217698778) you can remove it from here as well.